.github/workflows/mdns__host-tests.yml

```diff
@@ -10,27 +10,41 @@ on:
 jobs:
   host_test_mdns:
     if: contains(github.event.pull_request.labels.*.name, 'mdns') || github.event_name == 'push'
-    name: Host test
-    runs-on: ubuntu-22.04
-    container: espressif/idf:release-v5.1
+    name: Host test build
+    runs-on: ubuntu-20.04
```

Comment:
What was the reason to use ubuntu-20.04 instead of ubuntu-22.04 ?

Reply:
The issue is that `dig` command somehow "fails" in the newer container (probably some issue with the multicast address?), seems like we're getting this issue on master already: https://github.com/espressif/esp-protocols/actions/runs/10062355553/job/27843127946

I'll add a comment to the yaml file explaining it. Unfortunately I wasn't able to use any `espressif/idf` container, but had to start with clean slate image and setup everything from scratch.